build.gradle

```diff
@@ -24,7 +24,9 @@ ext {
 }
 
 dependencies {
-    testImplementation("org.testng:testng:6.14.2")
+    testImplementation("org.testng:testng:7.5") {
+        exclude group: "org.slf4j", module: "slf4j-log4j12"
```

Comment:
Is it possible to add a comment explaining why this dependency needs to be excluded (which tests are failing?) so that we can come back to it

Reply:
tests are not failing, I does not even pas the compile phase. I will add a comment

Reply:
@sbesson done